src/childinfo.c

```diff
@@ -69,18 +70,37 @@ void closeChildInfoPipe(void) {
 void sendChildInfoGeneric(childInfoType info_type, size_t keys, double progress, char *pname) {
     if (server.child_info_pipe[1] == -1) return;
 
-    child_info_data data = {0}; /* zero everything, including padding to sattisfy valgrind */
+    static ustime_t cow_updated = 0;
```

Comment:
I imagined that on each fork, the first sample of COW will always be emitted and will set the timing of the next ones. 
Considering the implementation, a (slightly hackish) way to do that would be to zero `cow_updated` when `info_type != CHILD_INFO_TYPE_CURRENT_INFO`. 

if a module would create a fork every 5 seconds, these can become heavy. 
But on the other hand, considering that we zero the metric each time we create a fork, if we don't do the above, these forks will usually not display the metric at all.

Reply:
Now that we may skip some of these reports I think the logic in rdb.c, aof.c and module.c to to send the initial report right away on startup (`keys_processed++ &1023`) is not very good since cow is likely to be 0 (we pay the price but gain nothing). 
Maybe we can figure out a way to modify the code in childinfo.c to send the initial report without cow, then after a second send one with cow, and then start the paste that's based in the cost?

Reply:
@oranagra I need to verify this, but I assume it will never be zero because we count private dirty, so all it takes is a single allocation on the child to increment that. Technically we don't count just COW but as the child is not expected to allocate anything significant, everything we see is practically COW overhead. [That makes me think that a misbehaving module can skew this]

Not sure I fully got your intention on the first comment, `cow_updated` is already initialized and since we never touch it in the parent all childs will spin up with it zero as it is.

Reply:
regarding cow_updated, the question is if we init it just once per run of redis-server, or each time we start a new fork.  I was in favor of inviting it, once per fork. 

Regarding zero COW, I didn't really mean 0, I mean that taking a reading right when the fork started, before any work was done, and then delaying the next reading by some 100 seconds, would be bad. In that case I would have preferred to take the initial reading after a second rather than right on the start.

Reply:
ohh, i missed the fact the static variable is not set in the parent process, so effectively cleared on every new fork.
i waive my argument about sending the initial cow report after the first second passed, hoping report resolution will be enough so that this doesn't matter.